src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,39 @@
+import { BiGithub, BiTwitter, BiLinkedin } from "../../assets/icons/Icons";
```

Comment:
Suggestion:- You can use absolute import here it will make your code look readable and cleaner.

Reply:
Forgot to change here. Thanks for pointing out.